roles/marathon/tasks/conf.yml

```diff
@@ -38,11 +38,30 @@
   tags:
     - marathon
 
+- name: create marathon directory
+  sudo: yes
+  file:
+    dest: /etc/marathon
+    state: directory
+    mode: 0755
```

Comment:
This next version of distributive-rpm will not contain Marathon checks in preparation to have them packaged with Marathon itself.